docs/source/learn/core_notebooks/pymc_aesara.ipynb

```diff
@@ -0,0 +1,2039 @@
+{
```

Comment:
<p>Image not rendering either. This should be added as an image file and commited to the core notebooks folder, then use markdown syntax to add the image with the alt text</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc/pull/5721/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
I am getting it as the aesara logo via github.